### PR TITLE
HSwish operation specification

### PR DESCRIPTION
**Pull request by mitruska**
Specification for HSwish (hard version of the swish) activation operation.
The HSwish operation is introduced in the [article](https://arxiv.org/pdf/1905.02244.pdf).

### CODE REVIEW

File: docs/ops/activation/HSwish_4.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+**Short description**: HSwish takes one input tensor and produces output tensor where the hard version of swish function is applied to the tensor elementwise.
+
+**Detailed description**: For each element from the input tensor calculates corresponding
+element in the output tensor with the following formula: 
+
+    \f[
+    HSwish(x) = x \frac{min(max(x + 3, 0), 6)}{6}
+    \f]
+
+The HSwish operation is introduced in the [article](https://arxiv.org/pdf/1905.02244.pdf).
```
Comment (postrational) [inline docs/ops/activation/HSwish_4.md:16]:
```suggestion
The HSwish operation is introduced in the following [article](https://arxiv.org/pdf/1905.02244.pdf).
```

File: docs/ops/activation/HSwish_4.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+## HSwish <a name="HSwish"></a> {#openvino_docs_ops_activation_HSwish_4}
```
Comment (lazarevevgeny) [inline docs/ops/activation/HSwish_4.md:1]: Please, update the docs/ops/opset4.md file.

**Comment by lazarevevgeny** [comment]
@dmitry-gorokhov, @vladimir-paramuzov, @antonvor, please, take a quick look at the HSwish operation specification. Does it correlate with what was implemented in CPU/GPU plugins?

File: docs/ops/activation/HSwish_4.md (hunk 2 above)
Comment (mitruska) [reply docs/ops/activation/HSwish_4.md:1]: Thanks, done

**Pull request merged**